packages/plugins/src/plugins/diagnostic-fix.plugin.ts

```diff
@@ -16,16 +16,19 @@ export class DiagnosticFixPlugin extends Plugin {
 
   /** @see https://github.com/microsoft/TypeScript/blob/main/src/compiler/diagnosticMessages.json for more codes */
   prioritizedCodes = [
-    80005, // 'require' call may be converted to an import
-    7005, // Variable implicitly has an ___ type
-    2339, // Property does not exist on type
-    7006, // Parameter implicitly has an ___ type
-    7008, // Member implicitly has an ___ type
-    80004, // JSDoc types may be moved to TypeScript types
-    90016, // Declare property
-    90035, // Declare private property
-    90053, // Declare a private field named
-    2525, // Initializer provides no value for this binding element and the binding element has no default value
+    80002, // convertFunctionToEs6Class
+    80005, // requireInTs
+    80004, // annotateWithTypeFromJSDoc
```

Comment:
Prioritized `types form jsDocs` over `from usage` because it seems easy to fix incorrect types in jsDocs or remove them to use types `from usage` instead of relying on the compiler.